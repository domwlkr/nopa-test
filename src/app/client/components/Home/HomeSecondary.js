import React from 'react';
import shapes from '../../../static/images/Shapes.svg';

const HomeSecondary = (props) => {
  return (
    <div className="secondary-content home-secondary">
      <div>
        <div>
          <h2>There's no such things as "one size fits all" finance.</h2>
          <p>We were founded to make money simple and fair for everyone: whether you're looking for a loan, or to get better rewards for your investments.</p>
        </div>
        <div>
          <img className="shapes" alt="Shapes" src={shapes} />
        </div>
      </div>
    </div>
  );
};

export default HomeSecondary;