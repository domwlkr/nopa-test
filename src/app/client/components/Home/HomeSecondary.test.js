import React from 'react';
import HomeSecondary from './HomeSecondary';
import renderer from 'react-test-renderer';

test('Renders correctly', () => {
  const homeSecondary = renderer.create(
    <HomeSecondary />
  ).toJSON();

  expect(homeSecondary).toMatchSnapshot();
});