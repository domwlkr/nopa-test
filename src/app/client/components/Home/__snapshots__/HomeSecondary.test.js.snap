// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="secondary-content home-secondary"
>
  <div>
    <div>
      <h2>
        There's no such things as "one size fits all" finance.
      </h2>
      <p>
        We were founded to make money simple and fair for everyone: whether you're looking for a loan, or to get better rewards for your investments.
      </p>
    </div>
    <div>
      <img
        alt="Shapes"
        className="shapes"
        src="test-file-stub"
      />
    </div>
  </div>
</div>
`;
